gallery/src/components/NxDropdownIconOnly/NxDropdownIconOnlyNavigationExample.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { faEllipsisV } from '@fortawesome/free-solid-svg-icons';
+import { NxDropdownIconOnly, useToggle, NxTextLink } from '@sonatype/react-shared-components';
+
+function NxDropdownIconOnlyNavigationExample() {
```

Comment:
Within the `NxDropdown` docs that I'm sure this was copied from, it always seemed weird that this was called the "navigation example".  I'm not sure that naming is super valid there nor here.

Reply:
> Within the `NxDropdown` docs that I'm sure this was copied from, it always seemed weird that this was called the "navigation example". I'm not sure that naming is super valid there nor here.

Per below I have consolidated the examples and tweaked the example descriptions: https://github.com/sonatype/sonatype-react-shared-components/pull/465/commits/7b3af6212d9d860691d028159220461d0d37da6e